src/SignhostAPIClient/Rest/DataObjects/Transaction.cs

```diff
@@ -7,9 +7,11 @@ public class Transaction
 	{
 		public string Id { get; set; }
 
+		public IDictionary<string, FileEntry> Files { get; set; }
+
 		public int Status { get; set; }
 
-		public bool Seal { get; set; }
+		public bool? Seal { get; set; }
```

Comment:
Although this is correct, most API integrators aren't aware of this property and they don't expect this behaviour.